src/app/services/client/client.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { CodeGovAPIClient } from '@code.gov/api-client';
+import { BehaviorSubject } from 'rxjs/';
+import { Observable } from 'rxjs/Observable';
+
+
+@Injectable()
+
+export class ClientService extends CodeGovAPIClient {
+
+  constructor() {
+
+    super({
+      base: 'https://code-api-staging.app.cloud.gov/api/0.1/',
```

Comment:
Can these be passed to the constructor and not be hardcoded values? Either as individual arguments or as an options object. These values can be our defaults.

Reply:
Good point.  I'm closing PR though. It's been superseded by other work.